docs/UserGuide.md

```diff
@@ -227,11 +227,18 @@ Adds a job application to the list.
 
 **Examples:**
 
-* `add c/Microsoft r/Software Engineer d/Nov 12 2022 1200 i/Technology s/Pending`
-  Adds a company called Microsoft, with the role Software Engineer in the technology industry,
-  deadline Nov 12 2022 1200 and status as pending.
-* `add c/Google r/Cleaner`
-  Adds a company called Google, with the role Cleaner and status `TO_BE_SUBMITTED`.
+***Successful Commands:***
```

Comment:
Thank you for the good example on the format of writing successful and failed command